lib/react.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const React = require('react');
+
+const Context = React.createContext();
+
+Context.displayName = 'StrangeMiddleEnd';
+
+exports.Provider = ({ middleEnd, children }) => {
```

Comment:
Giving this a function name will make it more introspectable via the dev tools.  Also sort of looking to react-redux as an example https://github.com/reduxjs/react-redux/blob/c581d480dd675f2645851fb006bef91aeb6ac24d/src/components/Provider.js#L6
```suggestion
exports.Provider = function Provider({ middleEnd, children }) {
```

Reply:
ahh, thanks for nabbing this, I'm so forgetful about component naming 🙏